src/js/controller/captions.js

```diff
@@ -35,6 +35,7 @@ const Captions = function(_model) {
                 /* eslint-disable no-loop-func */
                 const track = tracks[i];
                 if (_kindSupported(track.kind) && !_tracksById[track._id]) {
+                    track.sideloaded = true;
```

Comment:
do we still need this line? it should be covered by the previous changes right?

Reply:
where is that being set? The lined below depend on `track.sideloaded` being set to true at some point.

Reply:
https://github.com/jwplayer/jwplayer/pull/3683#discussion_r426621562

Reply:
The code that you pointed is not reached with this set up, so I had to add it here.

Reply:
ah! okay makes sense